build.gradle

```diff
@@ -372,8 +399,42 @@ task copyHtml(type: Copy) {
 // TODO: Temporary workaround for new exes / JRE bundling not quite being officially ready yet
 distZip.exclude("*.x*.exe")
 
+task unpackZipForNSIS(type: Copy) {
+    dependsOn distZip
+
+    def zipFile = file("${buildDir}/distributions/TerasologyLauncher.zip")
+    def tmpDir = file("${buildDir}/tmp/distNSIS")
+    from zipTree(zipFile)
+    into tmpDir
+}
+
+task renderNSISScript(type: Copy) {
+    dependsOn unpackZipForNSIS
+
+    from("win-setup/win-setup.nsi")
+    into("${buildDir}/tmp/distNSIS")
+    expand(appName: "TerasologyLauncher", guiName: "Terasology Launcher", publisher: orgName,
+      verString: displayVersion, verMajor: versionMajor, verMinor: versionMinor,
+      stringBundlesDir: "${projectDir}/win-setup/resources/strings", outDir: "${buildDir}/distributions",
+      licenseFile: "${projectDir}/LICENSE", minJREVersion: targetCompatibility.toString())
+}
+
+task copyNSISresources(type: Copy) {
+    from("win-setup/resources/images")
+    into("${buildDir}/tmp/distNSIS")
+}
+
+task distNSIS(type: Exec) {
+    dependsOn renderNSISScript
+    dependsOn copyNSISresources
+
+    executable "makensis"
```

Comment:
what happens if `makensis` is not installed? Can we fail gracefully in that case with a good error message?